assets/findify-recommendation.css

```diff
@@ -114,4 +109,21 @@
     width: 98%;
     height: 1.6em;
     white-space: nowrap;
+}
+
+.swiper-button-next,
+.swiper-button-prev {
+    background-color: #5396ED;
+    border-radius: 50%;
+    padding: 10px 20px;
+    color: white;
+    width: 38px !important;
+    height: 38px !important;
+}
```

Comment:
To have the bullets below the cards keeping the arrows centered put this:
top: calc(50% - 30px)

(30px as the padding-bottom on the wrapper)